scenariolib/visit.go

```diff
@@ -408,23 +404,32 @@ func (v *Visit) FindDocumentRankByTitle(toFind string) int {
 // WaitBetweenActions Wait a random or constant number of seconds between user actions
 func WaitBetweenActions(timeToWait int, isConstant bool) {
 	if isConstant {
-		time.Sleep((time.Duration(timeToWait*1000) + 500) * time.Second)
+		// Never faster than 500 Millisecond
+		time.Sleep(MaxDuration(time.Duration(timeToWait*1000)*time.Millisecond, 500*time.Millisecond))
 	} else {
 		// Failsafe to never go higher than 2 QPS per bot.
-		time.Sleep((time.Duration(rand.Intn(timeToWait*1000)) + 500) * time.Millisecond)
+		time.Sleep(MaxDuration(time.Duration(rand.Intn(timeToWait*1000))*time.Millisecond, 500*time.Millisecond))
```

Comment:
I find this line hard to read. Sleep of a Max of Time of a Randomize number. 
Maybe? 
```
var timeToWaitMs int = timeToWait*1000
if !isConstant {
  timeToWaitMs = rand.Intn(timeToWaitMs)
}
timeToWaitMs = Max(timeToWaitMs, 500)
time.Sleep( timeToWaitMs*time.Millisecond )
```